crypto_gcm.go

```diff
@@ -47,20 +47,22 @@ func (c *cryptoGCM) encrypt(pkt *recordLayer, raw []byte) ([]byte, error) {
 	payload := raw[recordLayerHeaderSize:]
 	raw = raw[:recordLayerHeaderSize]
 
-	nonce := append(append([]byte{}, c.localWriteIV[:4]...), make([]byte, 8)...)
+	nonce := make([]byte, 12)
```

Comment:
Mind making the 12 a constant? Beyond that great work! You are killing :)

Reply:
done!